public/app/features/alerting/unified/components/rule-editor/FolderAndGroup.tsx

```diff
@@ -392,8 +393,11 @@ function EvaluationGroupCreationModal({
         <form onSubmit={handleSubmit(() => onSubmit())}>
           <Field
             label={
-              <Label htmlFor={'group'} description="A group evaluates all its rules over the same evaluation interval.">
-                Evaluation group
+              <Label
+                htmlFor={evaluationGroupNameId}
```

Comment:
There was already another input rendered on the page with `id` of `group` - the evaluation picker. The label and input weren't linked together properly, and this is so I could select it by label text in the test